index.html

```diff
@@ -38,7 +38,7 @@ <h2 class="head">Contact Information</h2>
         </ul>
       </section>
       <section class="bookslistadd" class="content-section">
-        <h2 id="addbookinstruction">Add a New Book</h2>
+        <h2 id="addbookinstruction">Add New Book</h2>
```

Comment:
- [ ] (Optional) Upon first load, your app is showing two sections in one view, it is showing the add book section and the book list section kindly fix this issue for improved user experience

Your app on first load
![image](https://github.com/rmiyoyo/Awesome-Books-v2-With-ES6/assets/81932478/8180e4a6-479f-451a-bef8-4fcf485370e0)

#### Note that the undefined book objects are book objects stored in localstorage by other apps. Fixing that bug would also improve the user experience of your app

![image](https://github.com/rmiyoyo/Awesome-Books-v2-With-ES6/assets/81932478/8bdb1122-8988-47f7-a030-6e4d4b89138e)